lib/App/DuckPAN/DDG.pm

```diff
@@ -80,6 +81,26 @@ sub get_blocks_from_current_dir {
 
             # Display to the user when a class has been successfully loaded.
             $self->app->emit_debug(" - $class (" . $class->triggers_block_type . ")");
+
+			unless ($blocks_plugins{$class->triggers_block_type}) {
+				$blocks_plugins{$class->triggers_block_type} = [];
+			}
+			push @{$blocks_plugins{$class->triggers_block_type}}, $class;
+
+			# We could potentially do other IA-specific checks here
+			my $trigger_types = $class->get_triggers;
+
+			# Check for useless uppercase triggers so we can warn
+			UC_TRIGGER: for my $triggers (values %$trigger_types){
```

Comment:
We should ignore regex trigger types here because they're allowed to match on Uppercase letters. Only Words trigger IA's can't have uppercase.